Uplift/Views/Supporting/SlidingTabBarView.swift

```diff
@@ -68,10 +68,10 @@ extension SlidingTabBarView {
 
     /// Configuration for a tab bar.
     struct TabBarConfig {
-        var color: Color = Constants.Colors.black
-        var selectedFont: Font = Constants.Fonts.labelBold
+        var font: Font = Constants.Fonts.labelBold
+        var selectedColor: Color = Constants.Colors.black
```

Comment:
We should rename this to `selectedFontColor` to make it more clear and keep it parallel with the "Underline"